### PR TITLE
Reorganizing CCO manual upgrade requirements

### PR DESCRIPTION
**Pull request by jeana-redhat**
No Jira or BZ, groundwork for making related content better/possible.

Note: The titles "within a minor version" may seem in conflict with the caveat on this content that it is for upgrades "between minor versions", but this is the place users would go to find this, and phrasing will be covered as a part of later work on this book.

Previews:
- [Manually creating IAM for AWS](https://deploy-preview-41645--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#manually-create-iam_manually-creating-iam-aws): Upgrade steps removed, _Additional resources_ links to upgrade steps added
- [Manually creating IAM for Azure](https://deploy-preview-41645--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure/manually-creating-iam-azure.html#manually-create-iam_manually-creating-iam-azure): same
- [Manually creating IAM for GCP](https://deploy-preview-41645--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/manually-creating-iam-gcp.html#manually-create-iam_manually-creating-iam-gcp): same
- [Updating a cluster within a minor version using the web console](https://deploy-preview-41645--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-within-minor.html#manually-maintained-credentials-upgrade_updating-cluster-within-minor): Added upgrade steps and _Additional resources_ links to all three _Manually creating IAM_ procedures. Updated procedure to use `.Verification` heading.
- [Updating a cluster within a minor version using the CLI](https://deploy-preview-41645--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-cli.html#manually-maintained-credentials-upgrade_updating-cluster-cli): same

### CODE REVIEW

**Comment by sagidlow** [comment]
@LalatenduMohanty Could you look at the OTA portion of the PR?

**Comment by sagidlow** [comment]
Also tagging OTA QA @jiajliu

**Review by akhil-rane — COMMENTED** [review]
this looks good!

**Comment by jeana-redhat** [comment]
@lwan-wanglin This PR shouldn't really change anything, but I think it still should have your input. PTAL when you can.

**Comment by lwan-wanglin** [comment]
> @lwan-wanglin This PR shouldn't really change anything, but I think it still should have your input. PTAL when you can.

LGTM! :+1:

File: updating/updating-cluster-cli.adoc [hunk 1]
```diff
@@ -20,8 +20,8 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
...
-* If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being upgraded to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[_Upgrading an OpenShift Container Platform cluster configured for manual mode with STS_].
+* If your cluster uses manually maintained credentials, ensure that the Cloud Credential Operator (CCO) is in an upgradeable state. For more information, see xref:../updating/updating-cluster-cli.adoc#manually-maintained-credentials-upgrade_updating-cluster-cli[Upgrading clusters with manually maintained credentials].
```
Comment (jottofar) [inline updating/updating-cluster-cli.adoc:23]: Believe link is incorrect.

File: updating/updating-cluster-within-minor.adoc [hunk 2]
```diff
@@ -20,8 +20,8 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
...
-* If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being upgraded to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[_Upgrading an OpenShift Container Platform cluster configured for manual mode with STS_].
+* If your cluster uses manually maintained credentials, ensure that the Cloud Credential Operator (CCO) is in an upgradeable state. For more information, see xref:../updating/updating-cluster-within-minor.adoc#manually-maintained-credentials-upgrade_updating-cluster-within-minor[Upgrading clusters with manually maintained credentials].
```
Comment (jottofar) [inline updating/updating-cluster-within-minor.adoc:23]: Link incorrect here as well.

File: updating/updating-cluster-cli.adoc (hunk 1 above)
Comment (jeana-redhat) [reply updating/updating-cluster-cli.adoc:23]: Thanks, Jack. This is where I intended to link to, and it works when I click it. Do you think it should go somewhere else?

Comment (jeana-redhat) [reply updating/updating-cluster-cli.adoc:23]: Yeah, I linked to the anchor where the main text addition is. That link up top is directing them to the same anchor :)

**Comment by LalatenduMohanty** [comment]
lgtm

**Comment by jeana-redhat** [comment]
4.6-4.9 cherrypicks will have to be done manually, but also will not impede 4.10 work. Will pick up any backporting needed post 4.10 GA

**Pull request merged**